### ISSUE TITLE
Not possible to choose an option with empty value, because it used as placeholder

### ISSUE THREAD

**Issue by Fedik** [Choices-js/Choices#912]
**Describe the bug**
Since #617 is not possible to have an empty value in multi-select.

**To Reproduce**
Steps to reproduce the behavior.
Create a multiselect field with:
```
<select multiple>
  <option value="">All</option>
  <option value="1">1</option>
  <option value="2">2</option>
  <option value="3">3</option>
</select>
```

**Expected behavior**
You should be able to select "All", as it in native `<select>` behavior.

**Additional context**
Choices should not take an empty option as placeholder. If `placeholder` is invalid attribute (#447), then `data-placeholder` is pretty valid attribute

**Comment by ksouthworth** [comment]
I'm also dealing with this issue, does anyone have a solution or a workaround?

**Comment by Xon** [comment]
Implemented as part of #1166
